Test/Utils/ComputeNode/Service.ps1

```diff
@@ -0,0 +1,99 @@
+. $PSScriptRoot\..\..\PesterLogger\PesterLogger.ps1
+. $PSScriptRoot\..\..\..\CIScripts\Common\Invoke-NativeCommand.ps1
+. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testbed.ps1
+. $PSScriptRoot\Configuration.ps1
+
+function Install-ServiceWithNSSM {
+    Param (
+        [Parameter(Mandatory=$true)] $Session,
+        [Parameter(Mandatory=$true)] $ServiceName,
+        [Parameter(Mandatory=$true)] $ExecutablePath,
+        [Parameter(Mandatory=$false)] [string[]] $CommandLineParams = @()
+    )
+
+    $Output = Invoke-NativeCommand -Session $Session -ScriptBlock {
+            nssm install $using:ServiceName "$using:ExecutablePath" $using:CommandLineParams
+    } -AllowNonZero -CaptureOutput
+
+    $NSSMServiceAlreadyCreatedError = 5
+    if ($Output.ExitCode -eq 0) {
+        Write-Log $Output.Output
+    }
+    elseif ($Output.ExitCode -eq $NSSMServiceAlreadyCreatedError) {
+        Write-Log "$ServiceName service already created, continuing..."
+    }
+    else {
+        $ExceptionMessage = @"
+Unknown (wild) error appeared while creating $ServiceName service.
+ExitCode: $Output.ExitCode
+NSSM output: $Output.Output
+"@
+        throw [HardError]::new($ExceptionMessage)
+    }
+}
+
+function Remove-ServiceWithNSSM {
+    Param (
+        [Parameter(Mandatory=$true)] $Session,
+        [Parameter(Mandatory=$true)] $ServiceName
+    )
+
+    $Output = Invoke-NativeCommand -Session $Session {
+        nssm remove $using:ServiceName confirm
+    } -CaptureOutput
+
+    Write-Log $Output.Output
+}
+
+function Start-RemoteService {
+    Param (
+        [Parameter(Mandatory=$true)] $Session,
+        [Parameter(Mandatory=$true)] $ServiceName
+    )
+
+    Invoke-Command -Session $Session -ScriptBlock {
+        Start-Service $using:ServiceName
+    } | Out-Null
+}
+
+function Stop-RemoteService {
+    Param (
+        [Parameter(Mandatory=$true)] $Session,
+        [Parameter(Mandatory=$true)] $ServiceName
+    )
+
+    Invoke-Command -Session $Session -ScriptBlock {
+        Stop-Service $using:ServiceName -ErrorAction SilentlyContinue
```

Comment:
Why is it ok to `ErrorAction SilentlyContinue` but not in `Start-RemoteService`?